### PR TITLE
IGAPP-1096: Poi clustering native

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

Testing: Most poi markers can be found in "Landeshauptstadt München"

### CODE REVIEW

File: native/src/constants/layers.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  fontSizeLarge,
+  fontSizeSmall,
+  groupCount,
+  mapMarker,
+  PoiFeature,
+  textOffsetY,
+} from 'api-client'
+import { ThemeType } from 'build-configs/ThemeType'
+
+export const clusterLayer = (theme: ThemeType): CircleLayerProps => ({
```
Comment (steffenkleinle) [inline native/src/constants/layers.ts:15]: Styling differs between web and native, right? No way to make this more uniform/reuse stuff?

**Review by steffenkleinle — APPROVED** [review]
Tested on android, looks good :+1:

File: native/src/constants/layers.ts (hunk 1 above)
Comment (f1sh1918) [reply native/src/constants/layers.ts:15]: Well the libs using different interfaces for the layers. But the result should be quite same. I first thought that we can unify the layers but thats not possible unfortunately

**Review by sarahsporck — APPROVED** [review]
You were really fast on this one :)
Tested on the android emulator 👌🏼.

**Pull request merged**